### PR TITLE
Fix PPP icon not loading and update README.md

### PR DESCRIPTION
**Pull request by rehad-a**
-Fixes the problem with the nustracker icon not loading in the top of PPP pages
-Updates the description in README.md

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -32,7 +35,7 @@ While it has a GUI, most of the user interactions happen using a CLI (Command Li
 ------
 
 ### Getting Started With nustracker
-If you are interested in using nustracker, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
```
Comment (wowsiddanth) [inline docs/index.md:35]: Links don't work.

File: docs/index.md [hunk 2]
```diff
@@ -32,7 +35,7 @@ While it has a GUI, most of the user interactions happen using a CLI (Command Li
...
+If you are interested in using nustracker, head over to the [_Getting Started_ section of the **User Guide**](UserGuide.html#getting-started).
```
Comment (wowsiddanth) [inline docs/index.md:38]: Links don't work.

Comment (rehad-a) [reply docs/index.md]: Fixed 👍

**Review by Joel-Sung — APPROVED** [review]
LGTM.

**Pull request merged**